scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontend.java

```diff
@@ -209,6 +211,10 @@
 
     private it.sauronsoftware.cron4j.Scheduler metricsMonitorScheduler;
 
+    private AOSynchronization privateStore;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the "privateStore" field and declare it as a local variable in the relevant methods. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1450)

Reply:
Yes we can actually remove the two fields added in this class, or is this an active object trick to fool the GC?

Reply:
ok